gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -143,6 +143,7 @@ private ChannelAndRequestCounter buildManagedChannel() {
                 isNullOrEmpty(readOptions.getGrpcServerAddress())
                     ? DEFAULT_GCS_GRPC_SERVER_ADDRESS
                     : readOptions.getGrpcServerAddress())
+            .enableRetry()
```

Comment:
Are there any options that we can configure for retries? Like number of retries, retry delay, etc?

Reply:
This is configured in method getGrpcServiceConfig() down below. However without setting this, I believe the settings down there would be no-op after I looked through grpc-java impl.

Reply:
Cool!
Can we delete some high-level retries that we added in java code? Or they are still needed?

Reply:
Potentially yes. Now we are unclear about what error status write requests (that we added retries for), e.g. queryWriteStatus, startResumableWrite return upon errors. Our current grpc retry config only covers retries for error statuses UNAVAILABLE/RESOURCE_EXHAUSTED. I would rather remove any retries we have now or add it under grpc retry policy after doing some experiments.

Reply:
Ack.